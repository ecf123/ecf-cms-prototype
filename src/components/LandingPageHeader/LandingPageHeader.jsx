import "./LandingPageHeader.scss";


const LandingPageHeader = () => {
  return (
    <div>LandingPageHeader</div>
  )
}

export default LandingPageHeader