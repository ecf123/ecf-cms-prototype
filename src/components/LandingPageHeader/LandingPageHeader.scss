@use "../../styles/variables" as *;

.header {
    background-color: $color-blue-navy-dark;
    padding: 0.5rem;
}