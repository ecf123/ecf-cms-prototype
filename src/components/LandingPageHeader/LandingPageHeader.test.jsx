import { customRender } from "../../utils/testUtils";
import LandingPageHeader from "./LandingPageHeader";

it("Should render LandingPageHeader", () => {
  const { container } = customRender(<LandingPageHeader />);

  expect(container).toMatchSnapshot();
});
