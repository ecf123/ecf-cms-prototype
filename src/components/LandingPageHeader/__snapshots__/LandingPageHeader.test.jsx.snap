// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render LandingPageHeader 1`] = `
<div>
  <div
    class="header"
  >
    <img
      alt="logo"
      src="logo-dark-bg.svg"
    />
  </div>
</div>
`;
